src/eradius_proxy.erl

```diff
@@ -3,12 +3,18 @@
 -behaviour(eradius_server).
 -export([radius_request/3, validate_arguments/1]).
 
+-ifdef(TEST).
+-export([resolve_routes/4, validate_options/1, new_request/3,
+         get_key/4, strip/4]).
+-endif.
+
 -include("eradius_lib.hrl").
 -include("dictionary.hrl").
 
 -define(DEFAULT_TYPE, realm).
 -define(DEFAULT_STRIP, false).
 -define(DEFAULT_SEPARATOR, "@").
+
```

Comment:
Is there a point in keep the defaults for type, strip and separator as defines? There where used in the eunit test, but there is only one place remaining that is using them.

Reply:
should have read further, actually the DEFAULT_OPTIONS define is obsolete. The places where it is used could be replaced with ```[]```

Reply:
It seems that we cant put `[]` into `validate_options/1`. It needs all default options to validate. In the scope of this PR I would save the defines.